packages/server/graphql/mutations/removeAgendaItem.ts

```diff
@@ -44,10 +44,8 @@ export default {
       return standardError(new Error('Agenda item not found'), {userId: viewerId})
     }
     const filterFn = (stage: AgendaItemsStage) => stage.agendaItemId === agendaItemId
-    const meetingIds = await removeStagesFromMeetings(filterFn, teamId, dataLoader)
-    // safe to do so because we guarantee only 1 action meeting at the same time
-    const [meetingId] = meetingIds
-    const data = {agendaItem, meetingId}
+    await removeStagesFromMeetings(filterFn, teamId, dataLoader)
+    const data = {agendaItem, meetingId: agendaItem.meetingId}
```

Comment:
`const [meetingId] = meetingIds` this was causing to return wrong meeting id when there's more than one meeting active. I've changed it to return `meetingId` strictly related to `agendaItem`

Reply:
this makes a lot more sense :+1: